plugins/GSdx/GS.cpp

```diff
@@ -960,6 +961,23 @@ EXPORT_C GSgetTitleInfo2(char* dest, size_t length)
 	strcpy(dest, s.c_str());
 }
 
+EXPORT_C GSgetImageSize(int *image_width, int *image_height)
+{
+	if (!gsopen_done || s_gs == nullptr)
+		return;
+
+	int en0 = s_gs->IsEnabled(0);
+	int en1 = s_gs->IsEnabled(1);
+	if (!en0 && !en1)
+		return;
+
+	// Disregard the disabled circuit. With full boot the BIOS will fill the disabled circuit with its own data.
+	int i = en0 && !en1 ? 0 : en1 && !en0 ? 1 : -1;
```

Comment:
This could be simplified into ``int i = (en0 ^ en1) ? en1 : -1;``. Current one might require a bit more effort to be put to understand due to the redundancy. (A rather negligible amount really)

A comment indicating '-1` is for merged contexts might also be nice.